crates/pop-cli/src/commands/build/spec.rs

```diff
@@ -195,7 +199,9 @@ impl BuildSpecCommand {
 		let para_id = self.id.unwrap_or(DEFAULT_PARA_ID);
 		// Notify user in case we need to build the parachain project.
 		if !self.release {
-			cli.warning("NOTE: this command defaults to DEBUG builds for development chain types. Please use `--release` (or simply `-r` for a release build...)")?;
+			cli.warning(
+				"NOTE: this flag is deprecated, this command defaults to a release build builds.",
```

Comment:
```suggestion
				"NOTE: this flag is deprecated, this command defaults to a release build.",
```

Reply:
Some context in a comment as to why this is deprecated would be appreciated please.